modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/roundstartslime.dm

```diff
@@ -236,6 +237,26 @@
 			if(new_style)
 				alterer.facial_hairstyle = new_style
 				alterer.update_hair(is_creating = TRUE)
+		if("Hair Color")
+			var/hair_area = tgui_alert(alterer, "Select which color you would like to change", "Hair Color Alterations", list("Hairstyle", "Facial Hair", "Both"))
+			if(!hair_area)
+				return
+			var/new_hair_color = input(alterer, "Select your new hair color", "Hair Color Alterations", alterer.dna.features["mcolor"]) as color|null
+			if(!new_hair_color)
+				return
+
+			switch(hair_area)
+
+				if("Hairstyle")
+					alterer.hair_color = new_hair_color
+					alterer.update_hair(is_creating = TRUE)
```

Comment:
God I hate the usage of this proc, I really need to go and fix that upstream because Kapu is incredibly lazy.

Reply:
I've had to deal with this... Pretty much everywhere I went and I had to restrain myself from cursing out at Kapu in a comment before.